src/gfn/gym/hypergrid.py

```diff
@@ -199,6 +199,22 @@ def true_dist_pmf(self) -> torch.Tensor:
         true_dist /= true_dist.sum()
         return true_dist
 
+    def all_indices(self) -> List[Tuple[int, ...]]:
```

Comment:
this will likely need to get removed / merged with the similar functionality currently present in the `multinode` branch.